toplevel/topdirs.ml

```diff
@@ -728,53 +622,25 @@ let () =
 let () =
   reg_show_prim "show_class"
     (fun env loc id lid ->
-<<<<<<< HEAD
-       let path, desc_class = Env.lookup_class ~loc lid env in
-       let _path, desc_cltype = Env.lookup_cltype ~loc lid env in
-       let _path, typedcl = Env.lookup_type ~loc lid env in
-       let hash_typedcl = Env.find_hash_type path env in
```

Comment:
Intentional for this to be dropped? It seems reasonable to keep in to me, but I might be missing context.

Reply:
It is not dropped - only the `hash_typedcl` bit is dropped
This is due to https://github.com/ocaml/ocaml/commit/1632e9b2cfaee20d9921b9e2828abb7931a376ad